client/src/components/Masthead/Masthead.vue

```diff
@@ -50,8 +50,19 @@ const activeTab = ref(props.initialActiveTab);
 const extensionTabs = ref([]);
 const windowToggle = ref(false);
 
+let entryPointStore;
+const its_menu = reactive({
```

Comment:
```suggestion
const itsMenu = reactive({
```

Avoid snake case in js